src/plugins/mdn/mdnPlugin.js

```diff
@@ -70,15 +62,14 @@ module.exports = async function mdnPlugin(msg) {
   }
   msg.handling();
 
-  const suffix = slugify(words.slice(1));
```

Comment:
removed slugify because it make a link go to something different that what we were extracting data

Reply:
nvm, removed my changes